guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,91 @@
 {
```

Comment:
<p><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.toc.html#image-translation-models" target="_blank">Image Translation Models</a></p><ul><li><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.toc.html#cyclegan" target="_blank">CycleGAN</a></code></li></ul><p> </p><p>The similar bug is fixed in PixToPix model also. </p><p>PixToPix also had metrics missing https://github.com/ArcGIS/geosaurus/pull/5569. </p><p> </p><p>Rest of the notes look good. Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/902/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Thanks @priyankatuteja 
* added the information you provided to the release notes